src/Symfony/Component/Cache/Traits/ApcuTrait.php

```diff
@@ -71,9 +71,26 @@ protected function doHave($id)
      */
     protected function doClear($namespace)
     {
-        return isset($namespace[0]) && class_exists('APCuIterator', false) && ('cli' !== PHP_SAPI || ini_get('apc.enable_cli'))
```

Comment:
I think we should just remove the `false` from the `class_exists` check, to allow using the polyfill implementation of APCuIterator

Reply:
The polyfill [doesn't use autoload to declare the class](https://github.com/symfony/polyfill/blob/master/src/Apcu/bootstrap.php), so false or true wouldn't change anything.
Which means installing the polyfill is another way to fix this issue immediately yes.

Reply:
See https://github.com/symfony/symfony/pull/24011